src/content/docs/full-stack-observability/observe-everything/get-started/get-started-auto-telemetry-pixie

```diff
@@ -0,0 +1,88 @@
+---
+title: Get started with Auto-Telemetry with Pixie
+tags:
+  - Pixie Auto-Telemetry
+  - Agentless APM
```

Comment:
Since we're not using Agentless APM anywhere anymore, I think we can omit it here as well. I would add the following tags:
* Service monitoring
* Kubernetes
* eBPF

Reply:
thanks!